kolibri/plugins/device/assets/src/views/ManageContentPage/SelectTransferSourceModal/DriveList.vue

```diff
@@ -1,26 +1,29 @@
 <template>
 
   <div class="drive-list">
-    <div v-if="drives.length === 0">
-      <UiAlert
-        type="info"
-        :dismissible="false"
-      >
-        {{ noDrivesText }}
-      </UiAlert>
-    </div>
-
-    <div v-else>
-      <h2>{{ $tr('drivesFound') }}</h2>
-      <KRadioButton
-        v-for="drive in drives"
-        :key="drive.id"
-        :label="enabledDriveLabel(drive)"
-        :buttonValue="drive.id"
-        :currentValue="value"
-        @change="$emit('input', drive.id)"
-      />
-    </div>
+    <KRadioButtonGroup>
```

Comment:
There is no need to have `KRadioButtonGroup` wrap `UiAlert` and other content. Here, it'd be rather

```
<div v-else>
  <h2>
    {{ $tr('drivesFound') }}
  </h2>
  <KRadioButtonGroup>
    <KRadioButton
      v-for="drive in drives"
      :key="drive.id"
      :label="enabledDriveLabel(drive)"
      :buttonValue="drive.id"
      :currentValue="value"
      @change="$emit('input', drive.id)"
    />
</KRadioButtonGroup>
</div>